temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowInboundCallsInterceptor.java

```diff
@@ -82,4 +82,81 @@ public WorkflowOutput execute(WorkflowInput input) {
       workflowRunSpan.finish();
     }
   }
+
+  @Override
+  public void handleSignal(SignalInput input) {
+    Tracer tracer = options.getTracer();
+    SpanContext rootSpanContext =
+        contextAccessor.readSpanContextFromHeader(input.getHeader(), tracer);
+    Span workflowSignalSpan =
+        spanFactory
+            .createWorkflowSignalSpan(
```

Comment:
I didn't read this closely, but what many of our other SDKs do is make this span a child-of the original inbound workflow-level span and a follows-from the inbound signal-level span. Is that what's happening here?

Reply:
can you elaborate more on this (a link, doc or something related to other SDKs)?

Reply:
When you view the trace in many SDKs, a `HandleSignal`'s parent is the same as the `RunWorkflow`'s parent. And the `HandleSignal` is just linked via follows-from to the span in the header.